frontend/src/components/settings/SettingInput.tsx

```diff
@@ -35,10 +36,10 @@ const SettingInput: React.FC<RenderSettingInputProps> = ({
   isDisabled = () => false,
 }) => {
   const { t } = useTranslation();
-  const label = t(`label.${setting.label}`, {
+  const label = t(getNestedTranslation("label", setting.label), {
```

Comment:
I suggest we create a hook called useTranslate() in which we call t(getNestedTranslation(....))

```
const t = useTranslate() // and then use t("label", setting.label)
```

Reply:
Change applied
I'm thinking about the case when we use useTranslation and useTranslate hooks in the same file so we will get two returned function named t